src/xdocs/checks/coding/multiplestringliterals.xml

```diff
@@ -74,11 +74,11 @@
 public class MyClass {
   String a = &quot;StringContents&quot;;
   String a1 = &quot;unchecked&quot;;
-  @SuppressWarnings(&quot;unchecked&quot;) // OK, duplicate strings are ignored in annotations
```

Comment:
Please go through the issue #13213 description again.

Reply:
@MANISH-K-07  in line 77 comment making sense regarding duplicate string  why we need to remove that commnet?

Reply:
Exactly my question !!
Why did you remove such sensible coments ??

The issue only asks you to remove unnecessary '//ok' comments which don't have any explanation followed,  from `Input files`. Please find the right path to the suppressed file.